### ISSUE TITLE
Support only_if to set condition to run command

### ISSUE THREAD

**Issue by mizzy** [walter-cd/walter#84]
I'd like to run Walter on Wercker and my local machine with same pipeline.yml.
But I don't like to run some commands locally. So I've written the setting like this.

```yaml
pipeline:
  - stage_name: Put SSH public key
    stage_type: command
    command: if [ "$WERCKER" = "true" ]; then echo $DIGITALOCEAN_SSH_KEY_PUBLIC > $HOME/.ssh/id_rsa.pub; fi
```

This is so ugly. I'd like to write the setting like this.

```yaml
pipeline:
  - stage_name: Put SSH public key
    stage_type: command
    command: echo $DIGITALOCEAN_SSH_KEY_PUBLIC > $HOME/.ssh/id_rsa.pub
    only_if: test "$WERCKER" = "true"
```

`only_if` takes a shell command and if the command returns exit status other than 0, walter does not run this command.

**Comment by takahi-i** [comment]
Cool!

**Comment by takahi-i** [comment]
Fixed with #91
